src/Support/SelectFields.php

```diff
@@ -390,6 +366,116 @@ private static function isMongodbInstance(GraphqlType $parentType): bool
         return isset($parentType->config['model']) ? app($parentType->config['model']) instanceof $mongoType : false;
     }
 
+    /**
+     * @param array $queryArgs
+     * @param array $field
+     * @param GraphqlType $parentType
+     * @param array $select
+     * @param array $with
+     * @param mixed $ctx
+     * @param FieldDefinition $fieldObject
+     * @param string $key
+     * @param Closure|null $customQuery
+     */
+    protected static function handleInterfaceFields(
+        array $queryArgs,
+        $field,
+        GraphqlType $parentType,
+        array &$select,
+        array &$with,
+        $ctx,
+        $fieldObject,
+        $key,
+        $customQuery
+    ) {
+        $relationsKey = Arr::get($fieldObject->config, 'alias', $key);
+
+        $with[$relationsKey] = function ($query) use ($queryArgs, $field, $parentType, &$select, $ctx, $customQuery, $key, $fieldObject) {
+            $parentTable = self::isMongodbInstance($parentType) ? null : self::getTableNameFromParentType($parentType);
```

Comment:
This part is executed when a `Query` calls the `->with()` method of the model.

See it as a query like so:
```php
MyModel::with(function ($query) {
  // We are here
})->get();
```